pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -86,22 +92,23 @@ def gurobi_run(model_file, warmstart_file, soln_file, mipgap, options, suffixes)
     # key is specified, so you have to stare at the
     # output to see if it was accepted.
     for key, value in options.items():
-        # When options come from the pyomo command, all
-        # values are string types, so we try to cast
-        # them to a numeric value in the event that
-        # setting the parameter fails.
-        try:
-            model.setParam(key, value)
-        except TypeError:
-            # we place the exception handling for checking
-            # the cast of value to a float in another
-            # function so that we can simply call raise here
-            # instead of except TypeError as e / raise e,
-            # because the latter does not preserve the
-            # Gurobi stack trace
-            if not _is_numeric(value):
-                raise
-            model.setParam(key, float(value))
+        if key != 'ISV_Key':  # ISV_Key not supported by Gurobi
```

Comment:
Instead of wrapping the whole block in the "if", a more general and extensible approach would be to do:
```
if key == 'ISV_Key':
    continue
# When options come from the pyomo command, all
# ...
```

Reply:
Agreed. See 
https://github.com/Pyomo/pyomo/pull/720/commits/a406a368070304f111a5de98d855b3c680c99b21